### ISSUE TITLE
Cannot open custom dictionary file on windows

### ISSUE THREAD

**Issue by petrdanecek** [streetsidesoftware/vscode-spell-checker#413]
It seems that absolute path is not expanded right on Windows10 with RemoteSSH development.

VSCode settings:
```
    "cSpell.dictionaryDefinitions": [
        { "name": "itTerms", "path": "C:/Users/petr/VSCode/data/itTerms_cspell.txt"}
    ],
    "cSpell.dictionaries": [
        "itTerms"
    ],
```
Results in error:
```
[Error - 4:55:17 AM] 370	2020-01-13T09:55:17.585Z	validateTextDocument: {"errno":-2,"code":"ENOENT","syscall":"open","path":"/home/petr/projectHome/c:\\Users\\petr\\VSCode\\data\\itTerms_cspell.txt"}	
```

Note that /home/petr/projectHome is remote project folder on Linux server. It seems to me that project home is injected into absolute path used by cSpell to read the dictionary file. 
Note2: I tried variations of path like using backslash and backspaced backslash.

```
Version: 1.41.1
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:58:56.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.16299
```

**Comment by Jason3S** [comment]
@petrdanecek I am guessing you are referring to [Visual Studio Code Remote Development](https://code.visualstudio.com/docs/remote/faq).

The spell checker only supports custom dictionary files stored on the remote. Because it is running on the remote server, it has no access to the local files.

**Comment by petrdanecek** [comment]
Yup you are right, I'm referring to Visual Studio Code Remote Development.
I understand it is not the part of great spellChecker, but is there any way how to overcame this? I mean maybe some other plugin which copy the dictionary file to remote host.
I'm working with multiple remotes and it would be hard to keep all the cSpell dictionaries in sync.
I do use the VSCode configuration, but it gets not practical with the huge number of words added.

**Comment by Jason3S** [comment]
I have to think about it.
I'm working on a better way to support large word lists.

**Comment by petrdanecek** [comment]
Ok this is great news. Let me know if you need some testing support.
It would be also nice to have some way how to add the new words directly from editor to the dedicated word list. At the moment it is possible to add only to the vscode configuration and move these from configuration to the particular word list manually.

**Comment by janosh** [comment]
Being able to pipe new words from a remote host back to local custom dictionaries would be awesome! 🚀
